ISS/models/core_models.py

```diff
@@ -221,13 +222,27 @@ def can_auto_anonymize(self):
     @transaction.atomic
     def merge_into(self, other):
         """
-        Reassigns all the records that tie to this user to another one and
+        Reassigns all (<-- lies) the records that tie to this user to another one and
         disables this user afterwards.
         """
         
         Thread.objects.filter(author=self).update(author=other)
+
+        Post.objects.filter(author=self).update(posted_from='127.0.0.1')
         Post.objects.filter(author=self).update(author=other)
 
+        PostSnapshot.objects.filter(obsolesced_by=self).update(obsolescing_ip='127.0.0.1')
+        PostSnapshot.objects.filter(obsolesced_by=self).update(obsolesced_by=other)
```

Comment:
You can save a round trip to the database by updating both fields together.

```suggestion
        PostSnapshot.objects.filter(obsolesced_by=self).update(
          obsolescing_ip='127.0.0.1',
          obsolesced_by=other
        )
```